examples/build_all_docker.sh

```diff
@@ -16,7 +16,12 @@ for arg in "$@"; do
 done
 
 for d in "status-message/" $(ls -d */ | grep -v -e "status-message\/$"); do
-    (./build_docker.sh ${d%%/})
+    for directory in $(find $d -type d); do
+        if [ -d "$directory/src" ]; then
+            echo building $d;
+            (cd "$d"; cargo near build reproducible-wasm --no-locked; cp target/wasm32-unknown-unknown/release/*.wasm res/;);
```

Comment:
unfortunately, the output paths are incorrect. 
i suggest just removing all `res/` paths as it's not clear now, what they could be used for, 

and removing all `cp` commands too.